test/byContract/IexecPocoBoost/IexecPocoBoostDelegate.test.ts

```diff
@@ -690,15 +571,21 @@ describe('IexecPocoBoostDelegate', function () {
                 requester: requester.address,
             }).toObject();
             const erc1271Instance = await createFakeERC1271();
-            erc1271Instance.isValidSignature.returns('0x1626ba7e');
-            appInstance.owner.returns(erc1271Instance.address);
-            datasetInstance.owner.returns(erc1271Instance.address);
-            workerpoolInstance.owner.returns(erc1271Instance.address);
+            await IERC721__factory.connect(await iexecAccessor.appregistry(), appProvider)
+                .transferFrom(appProvider.address, erc1271Instance.address, appAddress)
+                .then((tx) => tx.wait());
+            await IERC721__factory.connect(await iexecAccessor.datasetregistry(), datasetProvider)
+                .transferFrom(datasetProvider.address, erc1271Instance.address, datasetAddress)
+                .then((tx) => tx.wait());
+            await IERC721__factory.connect(await iexecAccessor.workerpoolregistry(), scheduler)
+                .transferFrom(scheduler.address, erc1271Instance.address, workerpoolAddress)
+                .then((tx) => tx.wait());
             requestOrder.requester = erc1271Instance.address;
-            appOrder.sign = someSignature;
-            datasetOrder.sign = someSignature;
-            workerpoolOrder.sign = someSignature;
-            requestOrder.sign = someSignature;
+            const sign = ethers.utils.id('valid-signature');
```

Comment:
Why not use `someSignture` here too ?

Reply:
Here signature must be explicitly considered as valid:
https://github.com/iExecBlockchainComputing/PoCo/blob/29998c84bf6287b37b782541b573a1dc67dd99e6/contracts/tools/testing/ERC1271Mock.sol#L13